app/views/doc/_ext_books.erb

```diff
@@ -0,0 +1,97 @@
+<h2>Books</h2>
+<div class='two-column'>
+  <div class='column-left'>
+    <ul class='books-list'>
+      <li>
+        <a href="/book"><img src="/images/progit2.png" width="107" height="141" /></a>
+        <h4><a href="/book">Pro Git</a></h4>
+        <p class='description'>
+          By Scott Chacon and Ben Straub
+          <img class="creative-commons" src="/images/creative-commons@2x.png" width="115" height="28" />
+        </p>
+      </li>
+      <li>
+        <a href="https://www.packtpub.com/application-development/mastering-git"><img src="https://d1ldz4te4covpm.cloudfront.net/sites/default/files/imagecache/ppv4_main_book_cover/3754OS_3537_Mastering%20Git_0.jpg" width="107"></a>
+        <h4><a href="https://www.packtpub.com/application-development/mastering-git">Mastering Git</a></h4>
+        <p class='description'>
+          By Jakub Narębski
+        </p>
+      </li>
+      <li>
+        <a href="https://www.manning.com/mcquaid/?a_aid=MikeMcQuaid&a_bid=5688bbf4"><img src="https://www.manning.com/mcquaid/mcquaid_cover150.jpg" width="107"></a>
+        <h4><a href="https://www.manning.com/mcquaid/?a_aid=MikeMcQuaid&a_bid=5688bbf4">Git in Practice</a></h4>
```

Comment:
shouldn't the ampersands be escaped here?

Reply:
just copied from the existing list, and it's working on both versions. Why should it be escaped (URL encoded, or escaped due to .erb syntax?) ?